src/Main.ps1

```diff
@@ -1088,7 +1093,7 @@ function Invoke-Pester {
                 Invoke-PluginStep -Plugins $Plugins -Step Start -Context @{
                     Containers               = $containers
                     Configuration            = $pluginConfiguration
-                    GlobalPluginData         = $state.PluginData
+                    GlobalPluginData         = $pluginData
```

Comment:
`$state.PluginData` is not assigned `$pluginData` hashtable yet. That occurs in `Invoke-Test` below.